docs/development/testing_application_tests.rst

```diff
@@ -48,9 +48,16 @@ Or the app-staging VM:
 .. code:: sh
 
     vagrant ssh app-staging
-    sudo su www-data -s /bin/bash
+    sudo bash
     cd /var/www/securedrop
     pytest -v tests
+    chown -R www-data /var/lib/securedrop /var/www/securedrop
+
+.. note::
```

Comment:
Can we have this as an warning than normal note? That way people will notice more than a simple note.

Reply:
done